### ISSUE TITLE
Add the option to create Property Node methods with a Get/Set prefix (instead of Read/Write)

### ISSUE THREAD

**Issue by StefanCals** [opengds/OpenGDS#93]
When creating a new Property Node method, I can only select : 
- Read Property
- Write Property

![image](https://github.com/opengds/OpenGDS/assets/36990897/5a0ace5b-8635-45ab-933a-e405c3d7ab20)

Depending on what I chose here, my method will get the prefix "Read" or "Write".

I would like to see the option to have this prefix be "Get" or "Set".

Either the option to change it here : 
![image](https://github.com/opengds/OpenGDS/assets/36990897/ad5ea740-d49c-4f27-98d2-e28c020e024b)

or possibly a Boolean selector somewhere to change between Read/Write and Get/Set. 

----

I know this is personal preference and LabVIEW native also calls them Read/Write.
But I prefer to use the term Read\Write to the VIs that interact with instruments.
And prefer to use the term Get\Set to VIs that interact with a class (attribute accessors).

For example: 
The accessor name ‘Read Data’ will make me think that this VI will read Data from an Instrument.
So I prefer ‘Get Data’ instead, because it's stored in the attributes.

**Comment by StefanCals** [comment]
Even better would be if this options could be toggled from the GOOP4 Options in the GOOP4 configurations.

![image](https://github.com/opengds/OpenGDS/assets/36990897/9c2f8c93-8819-4b55-8abb-524b79e21705)

**Comment by MikaelHolmstrom** [comment]
I agree I also use Read and Write when talking to instruments and Get and Set when I only access the Instrument object's attribute And I most often just use Property Methods not Property Node Method.
And Yes I used the NI naming convention for this.
Okay it would be easy to add that extra option in the GOOP4 Config dialog

**Comment by MikaelHolmstrom** [comment]
Hi, we are working on some major clean up tasks in the code right now.
This is to disconnect the UML modeller from the OpenGDS base function, just to make it smaller so we can publish it on tools network so you can find it in VIPM.
BTW I see that you are not using the latest version, the config dialog looks like this now:
![image](https://github.com/opengds/OpenGDS/assets/6456318/912e21ae-dffd-477a-b377-0cc5a29d5c2d)
